concurrent_test.go

```diff
@@ -0,0 +1,92 @@
+package bbolt_test
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	bolt "go.etcd.io/bbolt"
+	"go.etcd.io/bbolt/internal/btesting"
+)
+
+func TestRepeatableRead(t *testing.T) {
```

Comment:
Nit: Have a `const` block here to define parameters for the test to see all of them together. Example:
``` 
const (
    readerInterval = 50 * time.Millisecond
    writerInterval = 10 * time.Millisecond
    testDuration = 30 * time.Second
)
```

Reply:
done.